tokens.scss

```diff
@@ -1,6 +1,6 @@
 ///
 /// Teikn v0.1.4
-/// Generated Thu Oct 10 2019 15:5:32
+/// Generated Mon Oct 28 2019 20:17:3
```

Comment:
weird that we use a generator for this when we already have to manually add the color in so many other places :)) 

but i did use `yarn generate`